### PR TITLE
Fix ccPolyline ctor keep_alive policy

### PR DESCRIPTION
**Pull request by tmontaigu**

### CODE REVIEW

**Review by dgirardeau — COMMENTED** [review]
I'm not able to really review this code, but at least I can approve ;)

File: script_examples/crop2d.py
Comment (tmontaigu) [inline script_examples/crop2d.py:1]: Should not be part of the commit actually, maybe juste the `addToDB(polyline)` can stay

**Comment by tmontaigu** [comment]
Windows CI is broken 🥶 , I need to fix it before merging this
